packages/firebase_database/lib/src/event.dart

```diff
@@ -34,3 +34,33 @@ class DataSnapshot {
   /// Returns the contents of this data snapshot as native types.
   dynamic get value => _data['value'];
 }
+
+class MutableData {
+  final Map<String, dynamic> _data;
+  MutableData._(this._data);
+
+  /// The key of the location that generated this DataSnapshot.
```

Comment:
Update dartdoc comment with the new class name

Reply:
Done!